src/hacbs/pages/PipelineRunPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import NamespacedPage from '../../components/NamespacedPage/NamespacedPage';
+import PageLayout from '../../components/PageLayout/PageLayout';
+import { useQuickstartCloseOnUnmount } from '../../hooks/useQuickstartCloseOnUnmount';
+import { getQueryArgument } from '../../shared/utils';
+import { PipelineRunDetailsView } from '../components/PipelineRunDetailsView/PipelineRunDetailsView';
+import PipelineRunsListView from '../components/PipelineRunListView/PipelineRunsListView';
+
+const PipelineRunPage = () => {
+  useQuickstartCloseOnUnmount();
+  const name = getQueryArgument('name');
+
+  return (
+    <NamespacedPage>
+      {name ? (
+        <React.Fragment>
+          <Helmet>
+            <title>PipelineRun Details Page</title>
+          </Helmet>
+          <PipelineRunDetailsView pipelineRunName={name} />
+        </React.Fragment>
+      ) : (
+        <React.Fragment>
+          <Helmet>
+            <title>PipelineRuns</title>
+          </Helmet>
+          <PageLayout title="PipelineRuns" description="PipelineRuns are listed here.">
+            <PageSection
+              padding={{ default: 'noPadding' }}
+              variant={PageSectionVariants.light}
+              isFilled
+            >
+              <PipelineRunsListView applicationName={name} />
+            </PageSection>
+          </PageLayout>
```

Comment:
We shouldn't be reading a full pipeline list page anymore right?

Ideally we don't use query params for the `name`. I think @rohitkrai03 mentioned he was going to change this at some point in app-studio.